slack-archive/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.output.ansi.enabled=ALWAYS
+#Mongo DB
+spring.data.mongodb.uri=mongodb://root:root@localhost:27017/slack-archive
+spring.data.mongodb.messages.collection=message-archive
```

Comment:
Maybe messages, such as channels? All collections in DB is archive and we have 'archive' in db name.

Reply:
Yes, you are right. It will be changed in next task. We need two collections for messages:
- for raw json mesages
- for parsed messages
and I did not bother with the collections names yet